deployment/roles/deploy/templates/worker-sts.yml.j2

```diff
@@ -77,4 +77,4 @@ spec:
         accessModes: ["ReadWriteOnce"]
         resources:
           requests:
-            storage: 8Gi
+            storage: 10Gi
```

Comment:
Shouldn't your "Clean up the entire working directory" commit fix the no-space-left issue?

Reply:
Not really, a few packages need even more space. I've seen [boost](https://git.centos.org/rpms/boost) eating up all the 8Gi, this is why I've increased the volume size to 10Gi.